druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/RocksDBLookupExtractor.java

```diff
@@ -63,8 +63,11 @@ public boolean canIterate() {
             RocksIterator it = db.newIterator();
             it.seekToFirst();
             while (it.isValid()) {
```

Comment:
can we limit the iterator to configured value ? Default is 100:

Reply:
Fixed